upgrade-guide.md

```diff
@@ -8,6 +8,7 @@ TODO
 
 ### Breaking Changes
 
+- **global_epsilon**: The `global_epsilon` value has been changed from 1E-8 to 1E-6 to better find numerical approximation. Also, `global_epsilon2` and `global_epsilon3` has been removed because they make no pratical sense. Use  `global_epsilon` instead.
```

Comment:
```suggestion
- **global_epsilon**: The `global_epsilon` value has been changed from 1E-8 to 1E-6 to better find numerical approximation. Also, `global_epsilon2` and `global_epsilon3` have been removed because they make no practical sense. Use  `global_epsilon` instead.
```